com.avaloq.tools.ddk.xtext.test.core/src/com/avaloq/tools/ddk/xtext/test/AbstractXtextMarkerBasedTest.java

```diff
@@ -623,7 +623,8 @@ public void setTestClassInvalid() {
    * @param testSource
    *          the test source, not {@code null}
    */
-  protected void beforeApplyAssertions(final XtextTestSource testSource) {}
+  protected void beforeApplyAssertions(final XtextTestSource testSource) {

```

Comment:
Should this declaration also change?

```suggestion
  protected void beforeApplyAssertions(XtextTestSource testSource) {
```